server/commands/deployments/DeployService.js

```diff
@@ -47,12 +86,37 @@ module.exports = function DeployServiceCommandHandler(command) {
     // Run asynchronously, we don't wait for deploy to finish intentionally
     deploy(deployment, destination, sourcePackage, command);
 
-    let accountName = deployment.accountName;
-    yield deploymentLogger.started(deployment, accountName);
+    yield deploymentLogger.started(deployment, deployment.accountName);
     return deployment;
   });
 };
 
+function checkAsgSchedule(asg) {
+  let state = getScheduleStateByAsg(asg);
+  if (state && state.toLowerCase() === 'off') {
+    throw new Error('The state of this ASG schedule is off.');
```

Comment:
Could we make this less ambiguous and just _"The ASG you are trying to deploy to is currently scheduled Off"_